syntaxes/http.tmLanguage

```diff
@@ -124,7 +124,7 @@
             </dict>
             <dict>
                 <key>match</key>
-                <string>^\s*#{1,}\s*(?:((@)name)\s+(\S+))?.*$</string>
+                <string>^\s*/{2,}\s*(?:((@)name)\s+(.+)|((@)note))?.*$</string>
```

Comment:
Please check the code that both `# @name` and `// @name` exist in the `http.tmLanguage`. You need to update both. And shall we create a separate match for `note`.